packages/indexer/src/indexer.test.ts

```diff
@@ -118,4 +118,11 @@ describe("Run Test", () => {
   });
 });
 
-const messages = generateMockMessages();
+const messages: MockStreamResponse[] = [...Array(10)].map((_, i) => ({
```

Comment:
I think we can use `generateMockMessages()` helper method here right?

Reply:
True now we have it!